diff -up systemd-stable-256/src/libsystemd/sd-journal/journal-file.c.13~ systemd-stable-256/src/libsystemd/sd-journal/journal-file.c
--- systemd-stable-256/src/libsystemd/sd-journal/journal-file.c.13~	2024-06-12 14:08:50.906352105 +0200
+++ systemd-stable-256/src/libsystemd/sd-journal/journal-file.c	2024-06-12 14:10:27.475378222 +0200
@@ -45,7 +45,7 @@
 #define DEFAULT_DATA_HASH_TABLE_SIZE (2047ULL*sizeof(HashItem))
 #define DEFAULT_FIELD_HASH_TABLE_SIZE (333ULL*sizeof(HashItem))
 
-#define DEFAULT_COMPRESS_THRESHOLD (512ULL)
+#define DEFAULT_COMPRESS_THRESHOLD (8192ULL)
 #define MIN_COMPRESS_THRESHOLD (8ULL)
 
 /* This is the minimum journal file size */
