diff -up systemd-245.20200407/src/journal/journald.c.13~ systemd-245.20200407/src/journal/journald.c
--- systemd-245.20200407/src/journal/journald.c.13~	2020-04-01 19:23:42.000000000 +0200
+++ systemd-245.20200407/src/journal/journald.c	2020-04-07 04:48:23.246854942 +0200
@@ -40,8 +40,6 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server, true);
-        server_flush_dev_kmsg(&server);
 
         if (server.namespace)
                 log_debug("systemd-journald running as PID "PID_FMT" for namespace '%s'.", getpid_cached(), server.namespace);
@@ -53,6 +51,9 @@ int main(int argc, char *argv[]) {
                               LOG_MESSAGE("Journal started"),
                               NULL);
 
+        server_flush_to_var(&server, true);
+        server_flush_dev_kmsg(&server);
+
         /* Make sure to send the usage message *after* flushing the
          * journal so entries from the runtime journals are ordered
          * before this message. See #4190 for some details. */
