diff -up systemd-258/src/libsystemd/sd-event/sd-event.c.16~ systemd-258/src/libsystemd/sd-event/sd-event.c
--- systemd-258/src/libsystemd/sd-event/sd-event.c.16~	2025-09-17 14:00:39.000000000 +0200
+++ systemd-258/src/libsystemd/sd-event/sd-event.c	2025-09-17 16:27:04.654016019 +0200
@@ -4831,9 +4831,18 @@ _public_ int sd_event_run(sd_event *e, u
         PROTECT_EVENT(e);
 
         r = sd_event_prepare(e);
-        if (r == 0)
+        if (r == 0) {
+                /* There was nothing? Then wait...
+                *
+                * ... but since there is no event pending, likely
+                * we'll be idle for a long time.  This is a good time
+                * to return any malloc() memory reserves to the
+                * operating system.
+                */
+               malloc_trim(0);
                 /* There was nothing? Then wait... */
                 r = sd_event_wait(e, timeout);
+        }
 
         if (e->profile_delays)
                 e->last_run_usec = now(CLOCK_MONOTONIC);
