diff -up systemd-251.20220617/src/core/locale-setup.c.18~ systemd-251.20220617/src/core/locale-setup.c
--- systemd-251.20220617/src/core/locale-setup.c.18~	2022-06-17 12:47:21.914252029 +0200
+++ systemd-251.20220617/src/core/locale-setup.c	2022-06-17 12:48:56.859746983 +0200
@@ -57,6 +57,12 @@ int locale_setup(char ***environment) {
                         log_warning_errno(r, "Failed to read /etc/locale.conf: %m");
         }
 
+	/* Nothing set... use some UTF-8 locale */
+	if (r <= 0) {
+		variables[VARIABLE_LANG] = strdup("en_US.UTF-8");
+	}
+	add = NULL;
+
         for (LocaleVariable i = 0; i < _VARIABLE_LC_MAX; i++) {
                 char *s;
 
