From 06372a72309235636f619e1b31aeef86414895fd Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Tue, 23 Jun 2015 11:50:44 +0100
Subject: [PATCH] locale-setup: set default locale to a unicode one.

---
 src/core/locale-setup.c | 5 +++++
 1 file changed, 5 insertions(+)

diff -Naur systemd-240.20190211/src/core/locale-setup.c systemd-240.20190211.tpg/src/core/locale-setup.c
--- systemd-240.20190211/src/core/locale-setup.c	2019-02-11 13:20:05.000000000 +0000
+++ systemd-240.20190211.tpg/src/core/locale-setup.c	2019-02-28 14:21:27.765000000 +0000
@@ -59,6 +59,11 @@
                         log_warning_errno(r, "Failed to read /etc/locale.conf: %m");
         }
 
+        /* Nothing set... use some UTF-8 locale */
+        if (r <= 0) {
+                variables[VARIABLE_LANG] = strdup("en_US.UTF-8");
+        }
+
         for (i = 0; i < _VARIABLE_LC_MAX; i++) {
                 char *s;
 
