diff -up systemd-258/units/systemd-udevd.service.in.18~ systemd-258/units/systemd-udevd.service.in
--- systemd-258/units/systemd-udevd.service.in.18~	2025-09-17 14:00:39.000000000 +0200
+++ systemd-258/units/systemd-udevd.service.in	2025-09-17 16:28:12.779910914 +0200
@@ -42,7 +42,7 @@ OOMScoreAdjust=-1000
 Sockets=systemd-udevd-control.socket systemd-udevd-kernel.socket systemd-udevd-varlink.socket
 Restart=always
 RestartSec=0
-ExecStart={{LIBEXECDIR}}/systemd-udevd
+ExecStart={{LIBEXECDIR}}/systemd-udevd --children-max=16
 FileDescriptorStoreMax=512
 FileDescriptorStorePreserve=yes
 KillMode=mixed
