diff -up systemd-stable-253/units/systemd-udevd.service.in.18~ systemd-stable-253/units/systemd-udevd.service.in
--- systemd-stable-253/units/systemd-udevd.service.in.18~	2023-02-16 11:50:57.730587839 +0100
+++ systemd-stable-253/units/systemd-udevd.service.in	2023-02-16 11:51:33.838086195 +0100
@@ -24,7 +24,7 @@ OOMScoreAdjust=-1000
 Sockets=systemd-udevd-control.socket systemd-udevd-kernel.socket
 Restart=always
 RestartSec=0
-ExecStart={{ROOTLIBEXECDIR}}/systemd-udevd
+ExecStart={{ROOTLIBEXECDIR}}/systemd-udevd --children-max=16
 KillMode=mixed
 TasksMax=infinity
 PrivateMounts=yes
