--- a/units/systemd-udevd.service.in	2026-09-22 17:23:04.093312374 +0200
+++ b/units/systemd-udevd.service.in	2026-09-22 17:23:04.093354074 +0200
@@ -42,7 +42,7 @@
 Sockets=systemd-udevd-kernel.socket systemd-udevd-varlink.socket
 Restart=always
 RestartSec=0
-ExecStart={{LIBEXECDIR}}/systemd-udevd
+ExecStart={{LIBEXECDIR}}/systemd-udevd --children-max=16
 FileDescriptorStoreMax=512
 FileDescriptorStorePreserve=yes
 KillMode=mixed
