diff -up systemd-245.20200407/units/systemd-udevd.service.in.18~ systemd-245.20200407/units/systemd-udevd.service.in
--- systemd-245.20200407/units/systemd-udevd.service.in.18~	2020-04-07 04:48:33.801974053 +0200
+++ systemd-245.20200407/units/systemd-udevd.service.in	2020-04-07 04:49:41.268733587 +0200
@@ -22,7 +22,7 @@ OOMScoreAdjust=-1000
 Sockets=systemd-udevd-control.socket systemd-udevd-kernel.socket
 Restart=always
 RestartSec=0
-ExecStart=@rootlibexecdir@/systemd-udevd
+ExecStart=@rootlibexecdir@/systemd-udevd --children-max=16
 ExecReload=udevadm control --reload --timeout 0
 KillMode=mixed
 TasksMax=infinity
