From ee960a27d8629d34c8e730dc28e95d0144d9a37c Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sat, 18 Jun 2016 17:33:33 +0000
Subject: [PATCH 24/39] more udev children workers

---
 units/systemd-udevd.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -Naur systemd-244.20191203/units/systemd-udevd.service.in systemd-244.20191203.tpg/units/systemd-udevd.service.in
--- systemd-244.20191203/units/systemd-udevd.service.in	2019-11-29 16:32:36.000000000 +0000
+++ systemd-244.20191203.tpg/units/systemd-udevd.service.in	2019-12-03 12:18:37.251022578 +0000
@@ -22,7 +22,7 @@
 Sockets=systemd-udevd-control.socket systemd-udevd-kernel.socket
 Restart=always
 RestartSec=0
-ExecStart=@rootlibexecdir@/systemd-udevd
+ExecStart=@rootlibexecdir@/systemd-udevd --children-max=16
 ExecReload=@rootbindir@/udevadm control --reload --timeout 0
 KillMode=mixed
 TasksMax=infinity
