diff -up systemd-261/src/libsystemd/sd-device/device-enumerator.c.21~ systemd-261/src/libsystemd/sd-device/device-enumerator.c
--- systemd-261/src/libsystemd/sd-device/device-enumerator.c.21~	2026-06-19 20:08:35.000000000 +0200
+++ systemd-261/src/libsystemd/sd-device/device-enumerator.c	2026-06-20 18:01:20.489353292 +0200
@@ -698,6 +698,23 @@ static bool relevant_sysfs_subdir(const
         return IN_SET(de->d_type, DT_DIR, DT_LNK);
 }
 
+static int acpi_status_disabled(char *sysfspath)
+{
+        FILE *file;
+        unsigned int i;
+        char newpath[4096];
+        char line[128];
+        sprintf(newpath, "%s/status", sysfspath);
+        file = fopen(newpath, "r");
+        if (!file)
+                return 0;
+        fclose(file);
+
+        if (strstr(sysfspath, "acpi") && strstr(sysfspath, "device:"))
+                return 1;
+        return 0;
+}
+
 static int enumerator_scan_dir_and_add_devices(
                 sd_device_enumerator *enumerator,
                 const char *basedir,
@@ -740,6 +757,9 @@ static int enumerator_scan_dir_and_add_d
 
                 (void) sprintf(syspath, "%s%s", path, de->d_name);
 
+                if (acpi_status_disabled(syspath))
+                        continue;
+
                 k = sd_device_new_from_syspath(&device, syspath);
                 if (k < 0) {
                         if (!ERRNO_IS_NEG_DEVICE_ABSENT(k))
