diff -up systemd-249.20210625/units/systemd-timesyncd.service.in.23~ systemd-249.20210625/units/systemd-timesyncd.service.in
--- systemd-249.20210625/units/systemd-timesyncd.service.in.23~	2021-06-25 17:31:52.332614810 +0200
+++ systemd-249.20210625/units/systemd-timesyncd.service.in	2021-06-25 17:33:20.218003540 +0200
@@ -51,7 +51,7 @@ StateDirectory=systemd/timesync
 SystemCallArchitectures=native
 SystemCallErrorNumber=EPERM
 SystemCallFilter=@system-service @clock
-Type=notify
+Type=simple
 User=systemd-timesync
 {{SERVICE_WATCHDOG}}
 
