From d53e8bc34944c931bb707e694bd1729e53fb8551 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Fri, 14 Jul 2017 11:35:13 -0700
Subject: [PATCH 29/45] Compile udev with -O3

udevd benefits from -O3 as it heavily affects boot time.
---

diff -Naur systemd-246.20200811/src/udev/meson.build systemd-246.20200811.tpg/src/udev/meson.build
--- systemd-246.20200811/src/udev/meson.build	2020-08-07 15:09:53.000000000 +0000
+++ systemd-246.20200811.tpg/src/udev/meson.build	2020-08-18 10:51:17.801045027 +0000
@@ -140,6 +140,7 @@
         disable_mempool_c,
         version : libudev_version,
         include_directories : includes,
+        c_args : ['-O3'],
         link_args : ['-shared',
                      '-Wl,--version-script=' + libudev_sym_path],
         link_with : [libsystemd_static, libshared_static],
@@ -156,7 +157,7 @@
         link_config_gperf_c,
         keyboard_keys_from_name_h,
         include_directories : libudev_core_includes,
-        c_args : ['-DLOG_REALM=LOG_REALM_UDEV'],
+        c_args : ['-DLOG_REALM=LOG_REALM_UDEV', '-O3'],
         link_with : udev_link_with,
         dependencies : [libblkid, libkmod])
 
