diff -Naur systemd-242/meson.build systemd-242.tpg/meson.build
--- systemd-242/meson.build	2019-04-15 12:24:21.436000000 +0000
+++ systemd-242.tpg/meson.build	2019-04-15 12:23:50.298000000 +0000
@@ -2662,7 +2662,7 @@
 exe = executable('systemd-udevd',
                  systemd_udevd_sources,
                  include_directories : includes,
-                 c_args : '-DLOG_REALM=LOG_REALM_UDEV',
+                 c_args : ['-DLOG_REALM=LOG_REALM_UDEV', '-O3'],
                  link_with : [libudev_core,
                               libsystemd_network,
                               libudev_static],
@@ -2679,7 +2679,7 @@
 
 exe = executable('udevadm',
                  udevadm_sources,
-                 c_args : '-DLOG_REALM=LOG_REALM_UDEV',
+                 c_args : ['-DLOG_REALM=LOG_REALM_UDEV', '-O3'],
                  include_directories : includes,
                  link_with : [libudev_core,
                               libsystemd_network,
diff -Naur systemd-242/src/udev/meson.build systemd-242.tpg/src/udev/meson.build
--- systemd-242/src/udev/meson.build	2019-04-15 12:24:54.947000000 +0000
+++ systemd-242.tpg/src/udev/meson.build	2019-04-15 12:22:06.420000000 +0000
@@ -179,7 +179,7 @@
         executable(prog[0].split('/')[0],
                    prog,
                    include_directories : includes,
-                   c_args : ['-DLOG_REALM=LOG_REALM_UDEV'],
+                   c_args : ['-DLOG_REALM=LOG_REALM_UDEV', '-O3'],
                    dependencies : [versiondep],
                    link_with : [libudev_static],
                    install_rpath : udev_rpath,
