diff -Naur systemd-252.20221101/rules.d/80-drivers.rules systemd-252.20221101.tpg/rules.d/80-drivers.rules
--- systemd-252.20221101/rules.d/80-drivers.rules	2022-10-31 18:59:18.000000000 +0000
+++ systemd-252.20221101.tpg/rules.d/80-drivers.rules	2022-11-01 10:43:36.149136163 +0000
@@ -2,6 +2,14 @@
 
 ACTION!="add", GOTO="drivers_end"
 
+# modprobe pci devices on cold plug except for:
+#  PCI_BASE_CLASS_STORAGE          0x01
+SUBSYSTEM=="pci", ENV{STARTUP}=="1", ATTR{class}=="0x01*", GOTO="drivers_end"
+
+# for display (0x03) devices
+# - skip loading if in initrd (i.e. no /initrd) and nokmsboot or failsafe is specified
+SUBSYSTEM=="pci", ATTR{class}=="0x03*", DRIVER!="?*", PROGRAM=="/bin/grep -qFw -e nokmsboot -e failsafe /proc/cmdline", GOTO="drivers_end"
+
 ENV{MODALIAS}=="?*", RUN{builtin}+="kmod load"
 SUBSYSTEM=="tifm", ENV{TIFM_CARD_TYPE}=="SD", RUN{builtin}+="kmod load tifm_sd"
 SUBSYSTEM=="tifm", ENV{TIFM_CARD_TYPE}=="MS", RUN{builtin}+="kmod load tifm_ms"
