diff -Naur systemd-244.20191203/rules.d/80-drivers.rules systemd-244.20191203.tpg/rules.d/80-drivers.rules
--- systemd-244.20191203/rules.d/80-drivers.rules	2019-11-29 16:32:36.000000000 +0000
+++ systemd-244.20191203.tpg/rules.d/80-drivers.rules	2019-12-03 12:12:28.177020450 +0000
@@ -2,6 +2,14 @@
 
 ACTION!="add", GOTO="drivers_end"
 
+# modprobe pci devices on cold plug except for:
+#  PCI_BASE_CLASS_STORAGE          0x01
+SUBSYSTEM=="pci", ENV{STARTUP}=="1", ATTR{class}=="0x01*", GOTO="drivers_end"
+
+# for display (0x03) devices
+# - skip loading if in initrd (i.e. no /initrd) and nokmsboot or failsafe is specified
+SUBSYSTEM=="pci", ATTR{class}=="0x03*", DRIVER!="?*", PROGRAM=="/bin/grep -qFw -e nokmsboot -e failsafe /proc/cmdline", GOTO="drivers_end"
+
 ENV{MODALIAS}=="?*", RUN{builtin}+="kmod load $env{MODALIAS}"
 SUBSYSTEM=="tifm", ENV{TIFM_CARD_TYPE}=="SD", RUN{builtin}+="kmod load tifm_sd"
 SUBSYSTEM=="tifm", ENV{TIFM_CARD_TYPE}=="MS", RUN{builtin}+="kmod load tifm_ms"
