diff -up systemd-240.20190114/src/udev/udevd.c.8~ systemd-240.20190114/src/udev/udevd.c
--- systemd-240.20190114/src/udev/udevd.c.8~	2019-01-14 01:25:37.048998327 +0100
+++ systemd-240.20190114/src/udev/udevd.c	2019-01-14 01:26:22.998704643 +0100
@@ -1823,7 +1823,7 @@ static int run(int argc, char *argv[]) {
         if (arg_daemonize) {
                 pid_t pid;
 
-                log_info("Starting version " PACKAGE_VERSION);
+                log_debug("Starting version " PACKAGE_VERSION);
 
                 /* connect /dev/null to stdin, stdout, stderr */
                 if (log_get_max_level() < LOG_DEBUG) {
