diff -up systemd-249.20210625/src/resolve/resolved.c.33~ systemd-249.20210625/src/resolve/resolved.c
--- systemd-249.20210625/src/resolve/resolved.c.33~	2021-06-25 17:36:15.194222833 +0200
+++ systemd-249.20210625/src/resolve/resolved.c	2021-06-25 17:36:15.203222806 +0200
@@ -58,6 +58,11 @@ static int run(int argc, char *argv[]) {
                 if (r < 0)
                         return log_error_errno(r, "Could not create runtime directory: %m");
 
+		r = symlink("../run/systemd/resolve/resolv.conf", "/etc/resolv.conf");
+		if (r < 0 && errno != EEXIST)
+			log_warning_errno(errno,
+					  "Could not create /etc/resolv.conf symlink: %m");
+
                 /* Drop privileges, but keep three caps. Note that we drop two of those too, later on (see below) */
                 r = drop_privileges(uid, gid,
                                     (UINT64_C(1) << CAP_NET_RAW)|          /* needed for SO_BINDTODEVICE */
