diff -up systemd-245.20200407/src/resolve/resolved.c.33~ systemd-245.20200407/src/resolve/resolved.c
--- systemd-245.20200407/src/resolve/resolved.c.33~	2020-04-07 04:50:18.255148753 +0200
+++ systemd-245.20200407/src/resolve/resolved.c	2020-04-07 04:53:21.323193090 +0200
@@ -50,6 +50,11 @@ static int run(int argc, char *argv[]) {
                 if (r < 0)
                         return log_error_errno(r, "Could not create runtime directory: %m");
 
+                r = symlink("../run/systemd/resolve/resolv.conf", "/etc/resolv.conf");
+                if (r < 0 && errno != EEXIST)
+                        log_warning_errno(errno,
+                                          "Could not create /etc/resolv.conf symlink: %m");
+
                 /* Drop privileges, but keep three caps. Note that we drop those too, later on (see below) */
                 r = drop_privileges(uid, gid,
                                     (UINT64_C(1) << CAP_NET_RAW)|          /* needed for SO_BINDTODEVICE */
diff -up systemd-245.20200407/tmpfiles.d/etc.conf.m4.33~ systemd-245.20200407/tmpfiles.d/etc.conf.m4
--- systemd-245.20200407/tmpfiles.d/etc.conf.m4.33~	2020-04-01 19:23:42.000000000 +0200
+++ systemd-245.20200407/tmpfiles.d/etc.conf.m4	2020-04-07 04:50:18.255148753 +0200
@@ -12,9 +12,6 @@ L+ /etc/mtab - - - - ../proc/self/mounts
 m4_ifdef(`HAVE_SMACK_RUN_LABEL',
 t /etc/mtab - - - - security.SMACK64=_
 )m4_dnl
-m4_ifdef(`ENABLE_RESOLVE',
-L! /etc/resolv.conf - - - - ../run/systemd/resolve/stub-resolv.conf
-)m4_dnl
 C! /etc/nsswitch.conf - - - -
 m4_ifdef(`HAVE_PAM',
 C! /etc/pam.d - - - -
