diff -up systemd-243.20190913/src/resolve/resolved.c.33~ systemd-243.20190913/src/resolve/resolved.c
--- systemd-243.20190913/src/resolve/resolved.c.33~	2019-09-13 01:57:24.683406430 +0200
+++ systemd-243.20190913/src/resolve/resolved.c	2019-09-13 01:57:24.698417251 +0200
@@ -49,6 +49,10 @@ static int run(int argc, char *argv[]) {
         /* Drop privileges, but only if we have been started as root. If we are not running as root we assume most
          * privileges are already dropped. */
         if (getuid() == 0) {
+                r = symlink("../run/systemd/resolve/resolv.conf", "/etc/resolv.conf");
+                if (r < 0 && errno != EEXIST)
+                        log_warning_errno(errno,
+                                          "Could not create /etc/resolv.conf symlink: %m");
 
                 /* Drop privileges, but keep three caps. Note that we drop those too, later on (see below) */
                 r = drop_privileges(uid, gid,
diff -up systemd-243.20190913/tmpfiles.d/etc.conf.m4.33~ systemd-243.20190913/tmpfiles.d/etc.conf.m4
--- systemd-243.20190913/tmpfiles.d/etc.conf.m4.33~	2019-09-13 01:57:24.698417251 +0200
+++ systemd-243.20190913/tmpfiles.d/etc.conf.m4	2019-09-13 01:58:21.684802161 +0200
@@ -12,9 +12,6 @@ L+ /etc/mtab - - - - ../proc/self/mounts
 m4_ifdef(`HAVE_SMACK_RUN_LABEL',
 t /etc/mtab - - - - security.SMACK64=_
 )m4_dnl
-m4_ifdef(`ENABLE_RESOLVE',
-L! /etc/resolv.conf - - - - ../run/systemd/resolve/stub-resolv.conf
-)m4_dnl
 C! /etc/nsswitch.conf - - - -
 m4_ifdef(`HAVE_PAM',
 C! /etc/pam.d - - - -
