diff -Naur systemd-246/src/core/macros.systemd.in systemd-246-p/src/core/macros.systemd.in
--- systemd-246/src/core/macros.systemd.in	2020-07-30 21:02:36.000000000 +0200
+++ systemd-246-p/src/core/macros.systemd.in	2020-08-31 23:10:21.951993361 +0200
@@ -45,9 +45,9 @@
 
 %systemd_post() \
 %{expand:%%{?__systemd_someargs_%#:%%__systemd_someargs_%# systemd_post}} \
-if [ $1 -eq 1 ] && [ -x @bindir@/systemctl ]; then \
+if [ $1 -eq 1 ] && [ -x @rootbindir@/systemctl ]; then \
         # Initial installation \
-        @bindir@/systemctl --no-reload preset %{?*} || : \
+        @rootbindir@/systemctl --no-reload preset %{?*} || : \
 fi \
 %{nil}
 
@@ -55,17 +55,17 @@
 
 %systemd_preun() \
 %{expand:%%{?__systemd_someargs_%#:%%__systemd_someargs_%# systemd_preun}} \
-if [ $1 -eq 0 ] && [ -x @bindir@/systemctl ]; then \
+if [ $1 -eq 0 ] && [ -x @rootbindir@/systemctl ]; then \
         # Package removal, not upgrade \
-        @bindir@/systemctl --no-reload disable --now %{?*} || : \
+        @rootbindir@/systemctl --no-reload disable --now %{?*} || : \
 fi \
 %{nil}
 
 %systemd_user_preun() \
 %{expand:%%{?__systemd_someargs_%#:%%__systemd_someargs_%# systemd_user_preun}} \
-if [ $1 -eq 0 ] && [ -x @bindir@/systemctl ]; then \
+if [ $1 -eq 0 ] && [ -x @rootbindir@/systemctl ]; then \
         # Package removal, not upgrade \
-        @bindir@/systemctl --global disable %{?*} || : \
+        @rootbindir@/systemctl --global disable %{?*} || : \
 fi \
 %{nil}
 
@@ -79,9 +79,9 @@
 
 %systemd_postun_with_restart() \
 %{expand:%%{?__systemd_someargs_%#:%%__systemd_someargs_%# systemd_postun_with_restart}} \
-if [ $1 -ge 1 ] && [ -x @bindir@/systemctl ]; then \
+if [ $1 -ge 1 ] && [ -x @rootbindir@/systemctl ]; then \
         # Package upgrade, not uninstall \
-        @bindir@/systemctl try-restart %{?*} || : \
+        @rootbindir@/systemctl try-restart %{?*} || : \
 fi \
 %{nil}
 
@@ -98,17 +98,17 @@
 # Deprecated. Use %tmpfiles_create_package instead
 %tmpfiles_create() \
 %{expand:%%{?__systemd_someargs_%#:%%__systemd_someargs_%# tmpfiles_create}} \
-[ -x @bindir@/systemd-tmpfiles ] && @bindir@/systemd-tmpfiles --create %{?*} || : \
+[ -x @rootbindir@/systemd-tmpfiles ] && @rootbindir@/systemd-tmpfiles --create %{?*} || : \
 %{nil}
 
 # Deprecated. Use %sysusers_create_package instead
 %sysusers_create() \
 %{expand:%%{?__systemd_someargs_%#:%%__systemd_someargs_%# sysusers_create}} \
-[ -x @bindir@/systemd-sysusers ] && @bindir@/systemd-sysusers %{?*} || : \
+[ -x @rootbindir@/systemd-sysusers ] && @rootbindir@/systemd-sysusers %{?*} || : \
 %{nil}
 
 %sysusers_create_inline() \
-[ -x @bindir@/systemd-sysusers ] && @bindir@/systemd-sysusers - <<SYSTEMD_INLINE_EOF || : \
+[ -x @rootbindir@/systemd-sysusers ] && @rootbindir@/systemd-sysusers - <<SYSTEMD_INLINE_EOF || : \
 %{?*} \
 SYSTEMD_INLINE_EOF\
 %{nil}
