diff -up systemd-stable-254.3/units/systemd-firstboot.service.7~ systemd-stable-254.3/units/systemd-firstboot.service
--- systemd-stable-254.3/units/systemd-firstboot.service.7~	2023-09-11 17:05:18.698078455 +0200
+++ systemd-stable-254.3/units/systemd-firstboot.service	2023-09-11 17:11:50.285275747 +0200
@@ -13,6 +13,7 @@ Documentation=man:systemd-firstboot(1)
 
 ConditionPathIsReadWrite=/etc
 ConditionFirstBoot=yes
+ConditionVirtualization=!container
 
 DefaultDependencies=no
 # This service may need to write to the file system:
