diff -Naur a/src/journal/journald-server.c b/src/journal/journald-server.c
--- a/src/journal/journald-server.c	2021-03-12 17:25:58.000000000 +0000
+++ b/src/journal/journald-server.c	2021-03-13 11:58:19.971382038 +0000
@@ -2190,7 +2190,7 @@
                 .compress.threshold_bytes = UINT64_MAX,
                 .seal = true,
 
-                .set_audit = true,
+                .set_audit = false,
 
                 .watchdog_usec = USEC_INFINITY,
 
diff -Naur a/src/journal/journald.conf b/src/journal/journald.conf
--- a/src/journal/journald.conf	2021-03-12 17:25:58.000000000 +0000
+++ b/src/journal/journald.conf	2021-03-13 11:57:46.176388276 +0000
@@ -42,4 +42,4 @@
 #MaxLevelWall=emerg
 #LineMax=48K
 #ReadKMsg=yes
-#Audit=yes
+Audit=no
