diff -up systemd-stable-256/src/journal/journald-server.c.6~ systemd-stable-256/src/journal/journald-server.c
--- systemd-stable-256/src/journal/journald-server.c.6~	2024-06-11 23:42:06.000000000 +0200
+++ systemd-stable-256/src/journal/journald-server.c	2024-06-12 14:08:00.809819448 +0200
@@ -2596,7 +2596,7 @@ int server_new(Server **ret) {
                 .compress.threshold_bytes = UINT64_MAX,
                 .seal = true,
 
-                .set_audit = true,
+                .set_audit = false,
 
                 .watchdog_usec = USEC_INFINITY,
 
diff -up systemd-stable-256/src/journal/journald.conf.6~ systemd-stable-256/src/journal/journald.conf
--- systemd-stable-256/src/journal/journald.conf.6~	2024-06-12 14:08:00.805819407 +0200
+++ systemd-stable-256/src/journal/journald.conf	2024-06-12 14:08:29.398123447 +0200
@@ -47,4 +47,4 @@ RuntimeMaxUse=150M
 #MaxLevelSocket=debug
 #LineMax=48K
 #ReadKMsg=yes
-#Audit=yes
+Audit=no
