diff -up systemd-247.20201128/units/systemd-firstboot.service.6~ systemd-247.20201128/units/systemd-firstboot.service
--- systemd-247.20201128/units/systemd-firstboot.service.6~	2020-11-28 18:11:28.238796000 +0100
+++ systemd-247.20201128/units/systemd-firstboot.service	2020-11-28 18:12:02.790529666 +0100
@@ -17,6 +17,7 @@ Before=systemd-sysusers.service sysinit.
 Wants=first-boot-complete.target
 ConditionPathIsReadWrite=/etc
 ConditionFirstBoot=yes
+ConditionVirtualization=!container
 
 [Service]
 Type=oneshot
