diff -up systemd-258/units/systemd-fsck-root.service.in.8~ systemd-258/units/systemd-fsck-root.service.in
--- systemd-258/units/systemd-fsck-root.service.in.8~	2025-09-17 14:00:39.000000000 +0200
+++ systemd-258/units/systemd-fsck-root.service.in	2025-09-17 16:24:29.413602942 +0200
@@ -23,3 +23,5 @@ RemainAfterExit=yes
 ExecStart={{LIBEXECDIR}}/systemd-fsck
 TimeoutSec=infinity
 ImportCredential=fsck.*
+StandardOutput=null
+StandardError=journal+console
diff -up systemd-258/units/systemd-fsck@.service.in.8~ systemd-258/units/systemd-fsck@.service.in
--- systemd-258/units/systemd-fsck@.service.in.8~	2025-09-17 14:00:39.000000000 +0200
+++ systemd-258/units/systemd-fsck@.service.in	2025-09-17 16:24:53.503722417 +0200
@@ -23,3 +23,5 @@ RemainAfterExit=yes
 ExecStart={{LIBEXECDIR}}/systemd-fsck %f
 TimeoutSec=infinity
 ImportCredential=fsck.*
+StandardOutput=null
+StandardError=journal+console
