diff -up systemd-249.20210625/src/rpm/macros.systemd.in.omv~ systemd-249.20210625/src/rpm/macros.systemd.in
--- systemd-249.20210625/src/rpm/macros.systemd.in.omv~	2021-06-30 05:01:58.340935200 +0200
+++ systemd-249.20210625/src/rpm/macros.systemd.in	2021-06-30 16:44:25.030329892 +0200
@@ -136,7 +136,7 @@ SYSTEMD_INLINE_EOF\
 %sysusers_create_package() \
 %{expand:%%{?!__systemd_twoargs_%#:%%{error:The %%%%sysusers_create_package macro requires two arguments}}} \
 systemd-sysusers --replace=%_sysusersdir/%1.conf - <<SYSTEMD_INLINE_EOF || : \
-%(cat %2) \
+%(sed -e 's,@LIBDIR@,%{_libdir},g;s,@LIBEXECDIR@,%{_libexecdir},g' %2) \
 SYSTEMD_INLINE_EOF\
 %{nil}
 
