diff -up systemd-249.20210625/src/boot/efi/meson.build.omv~ systemd-249.20210625/src/boot/efi/meson.build
--- systemd-249.20210625/src/boot/efi/meson.build.omv~	2021-06-25 17:46:50.343244024 +0200
+++ systemd-249.20210625/src/boot/efi/meson.build	2021-06-25 17:58:02.529412391 +0200
@@ -175,19 +175,15 @@ if have_gnu_efi
 endif
 
 if have_gnu_efi
-        compile_args = cc.get_supported_arguments(
-                basic_disabled_warnings +
-                possible_common_cc_flags + [
-                        '-ffreestanding',
-                        '-fno-stack-protector',
-                        '-fno-strict-aliasing',
-                        '-fpic',
-                        '-fshort-wchar',
-                        '-Wall',
-                        '-Wextra',
-                        '-Wsign-compare',
-                ]
-        ) + [
+        compile_args = [
+                '-ffreestanding',
+                '-fno-stack-protector',
+                '-fno-strict-aliasing',
+                '-fpic',
+                '-fshort-wchar',
+                '-Wall',
+                '-Wextra',
+                '-Wsign-compare',
                 '-nostdlib',
                 '-std=gnu99',
                 '-isystem', efi_incdir,
