diff -up systemd-243.20190913/units/systemd-random-seed.service.in.omv~ systemd-243.20190913/units/systemd-random-seed.service.in
--- systemd-243.20190913/units/systemd-random-seed.service.in.omv~	2019-09-15 14:01:05.152772560 +0200
+++ systemd-243.20190913/units/systemd-random-seed.service.in	2019-09-15 14:03:17.719446890 +0200
@@ -27,4 +27,6 @@ ExecStop=@rootlibexecdir@/systemd-random
 # used as ordering barrier for service that require an initialized entropy
 # pool. Since initialization can take a while on entropy-starved systems, let's
 # increase the time-out substantially here.
-TimeoutSec=10min
+# But since a typical user will power-cycle if booting takes 10 minutes
+# (which will just make things worse), let's not slow down bootup too badly.
+TimeoutSec=10sec
