diff -up systemd-250.20220315/src/libsystemd/disable-mempool.c.32~ systemd-250.20220315/src/libsystemd/disable-mempool.c
--- systemd-250.20220315/src/libsystemd/disable-mempool.c.32~	2022-03-11 14:37:06.000000000 +0100
+++ systemd-250.20220315/src/libsystemd/disable-mempool.c	2022-03-15 18:51:22.352968749 +0100
@@ -2,4 +2,4 @@
 
 #include "mempool.h"
 
-const bool mempool_use_allowed = false;
+__attribute__((used)) const bool mempool_use_allowed = false;
diff -up systemd-250.20220315/src/shared/enable-mempool.c.32~ systemd-250.20220315/src/shared/enable-mempool.c
--- systemd-250.20220315/src/shared/enable-mempool.c.32~	2022-03-11 14:37:06.000000000 +0100
+++ systemd-250.20220315/src/shared/enable-mempool.c	2022-03-15 18:51:22.353968724 +0100
@@ -2,4 +2,4 @@
 
 #include "mempool.h"
 
-const bool mempool_use_allowed = true;
+__attribute__((used)) const bool mempool_use_allowed = true;
diff -up systemd-250.20220315/src/test/test-set.c.32~ systemd-250.20220315/src/test/test-set.c
--- systemd-250.20220315/src/test/test-set.c.32~	2022-03-15 18:51:22.353968724 +0100
+++ systemd-250.20220315/src/test/test-set.c	2022-03-15 18:55:08.556098747 +0100
@@ -5,7 +5,7 @@
 #include "strv.h"
 #include "tests.h"
 
-const bool mempool_use_allowed = VALGRIND;
+__attribute__((used)) const bool mempool_use_allowed = VALGRIND;
 
 TEST(set_steal_first) {
         _cleanup_set_free_ Set *m = NULL;
