diff -up systemd-246.20200806/src/import/import-common.c.34~ systemd-246.20200806/src/import/import-common.c
--- systemd-246.20200806/src/import/import-common.c.34~	2020-08-06 23:02:11.227462692 +0200
+++ systemd-246.20200806/src/import/import-common.c	2020-08-06 23:03:02.525124891 +0200
@@ -85,7 +85,6 @@ int import_fork_tar_x(const char *path,
                        "-C", path,
                        "-px",
                        "--xattrs",
-                       "--xattrs-include=*",
                        use_selinux ? "--selinux" : "--no-selinux",
                        NULL
                 };
@@ -156,7 +155,6 @@ int import_fork_tar_c(const char *path,
                         "-C", path,
                         "-c",
                         "--xattrs",
-                        "--xattrs-include=*",
                        use_selinux ? "--selinux" : "--no-selinux",
                         ".",
                         NULL
