diff -up systemd-stable-256/src/vmspawn/vmspawn-util.h.omv~ systemd-stable-256/src/vmspawn/vmspawn-util.h
--- systemd-stable-256/src/vmspawn/vmspawn-util.h.omv~	2024-07-14 19:20:54.237730629 +0100
+++ systemd-stable-256/src/vmspawn/vmspawn-util.h	2024-07-14 19:25:04.148851289 +0100
@@ -34,7 +34,7 @@
 
 #if defined(__x86_64__) || defined(__i386__)
 #  define QEMU_MACHINE_TYPE "q35"
-#elif defined(__arm__) || defined(__aarch64__)
+#elif defined(__arm__) || defined(__aarch64__) || defined(__riscv)
 #  define QEMU_MACHINE_TYPE "virt"
 #elif defined(__s390__) || defined(__s390x__)
 #  define QEMU_MACHINE_TYPE "s390-ccw-virtio"
