diff -up systemd-stable-254.3/units/systemd-fsck-root.service.in.6~ systemd-stable-254.3/units/systemd-fsck-root.service.in
--- systemd-stable-254.3/units/systemd-fsck-root.service.in.6~	2023-09-11 17:01:28.116493331 +0200
+++ systemd-stable-254.3/units/systemd-fsck-root.service.in	2023-09-11 17:04:15.012375484 +0200
@@ -22,3 +22,5 @@ Type=oneshot
 RemainAfterExit=yes
 ExecStart={{ROOTLIBEXECDIR}}/systemd-fsck
 TimeoutSec=infinity
+StandardOutput=null
+StandardError=journal+console
diff -up systemd-stable-254.3/units/systemd-fsck@.service.in.6~ systemd-stable-254.3/units/systemd-fsck@.service.in
--- systemd-stable-254.3/units/systemd-fsck@.service.in.6~	2023-09-11 17:01:28.116493331 +0200
+++ systemd-stable-254.3/units/systemd-fsck@.service.in	2023-09-11 17:04:45.601714052 +0200
@@ -21,3 +21,5 @@ Type=oneshot
 RemainAfterExit=yes
 ExecStart={{ROOTLIBEXECDIR}}/systemd-fsck %f
 TimeoutSec=infinity
+StandardOutput=null
+StandardError=journal+console
