diff -up systemd-260.1/src/sbsign/sbsign.c.omv~ systemd-260.1/src/sbsign/sbsign.c
--- systemd-260.1/src/sbsign/sbsign.c.omv~	2026-05-24 21:32:30.934697882 +0200
+++ systemd-260.1/src/sbsign/sbsign.c	2026-05-24 21:34:03.220651517 +0200
@@ -262,8 +262,9 @@ static int spc_indirect_data_content_new
                 return log_error_errno(SYNTHETIC_ERRNO(EIO), "Failed to get SpcPeImageData object: %s",
                                        ERR_error_string(ERR_get_error(), NULL));
 
-        idc->data->value->value.sequence->data = TAKE_PTR(peidraw);
-        idc->data->value->value.sequence->length = peidrawsz;
+	ASN1_STRING *seq = idc->data->value->value.sequence;
+	if(seq)
+		ASN1_STRING_set0(seq, TAKE_PTR(peidraw), peidrawsz);
         idc->messageDigest->digestAlgorithm->algorithm = OBJ_nid2obj(NID_sha256);
         if (!idc->messageDigest->digestAlgorithm->algorithm)
                 return log_error_errno(SYNTHETIC_ERRNO(EIO), "Failed to get SHA256 object: %s",
diff -up systemd-260.1/src/shared/pkcs11-util.c.omv~ systemd-260.1/src/shared/pkcs11-util.c
--- systemd-260.1/src/shared/pkcs11-util.c.omv~	2026-05-24 21:30:13.281095790 +0200
+++ systemd-260.1/src/shared/pkcs11-util.c	2026-05-24 21:32:04.006639490 +0200
@@ -554,7 +554,7 @@ int pkcs11_token_read_public_key(
                         return log_debug_errno(SYNTHETIC_ERRNO(EIO), "Failed to init an EVP_PKEY_CTX for EC.");
 
                 OSSL_PARAM ec_params[8] = {
-                        OSSL_PARAM_octet_string(OSSL_PKEY_PARAM_PUB_KEY, os->data, os->length)
+                        OSSL_PARAM_octet_string(OSSL_PKEY_PARAM_PUB_KEY, (void*)ASN1_STRING_get0_data(os), ASN1_STRING_length(os))
                 };
 
                 _cleanup_free_ void *order = NULL, *p = NULL, *a = NULL, *b = NULL, *generator = NULL;
@@ -691,7 +691,7 @@ int pkcs11_token_read_x509_certificate(
         if (!x509)
                 return log_debug_errno(SYNTHETIC_ERRNO(EBADMSG), "Failed to parse X.509 certificate.");
 
-        name = X509_get_subject_name(x509);
+        name = (X509_NAME*)X509_get_subject_name(x509);
         if (!name)
                 return log_debug_errno(SYNTHETIC_ERRNO(EBADMSG), "Failed to acquire X.509 subject name.");
 
