diff -up systemd-248.20210409/src/shared/openssl-util.c.omv~ systemd-248.20210409/src/shared/openssl-util.c
--- systemd-248.20210409/src/shared/openssl-util.c.omv~	2021-04-09 22:02:16.944556523 +0200
+++ systemd-248.20210409/src/shared/openssl-util.c	2021-04-09 22:02:25.304519384 +0200
@@ -46,7 +46,7 @@ int rsa_pkey_to_suitable_key_size(
                 size_t *ret_suitable_key_size) {
 
         size_t suitable_key_size;
-        RSA *rsa;
+        const RSA *rsa;
         int bits;
 
         assert_se(pkey);
