diff -up systemd-stable-255/units/systemd-fsck-root.service.in.7~ systemd-stable-255/units/systemd-fsck-root.service.in
--- systemd-stable-255/units/systemd-fsck-root.service.in.7~	2023-12-08 14:11:50.229849785 +0100
+++ systemd-stable-255/units/systemd-fsck-root.service.in	2023-12-08 14:12:55.950515316 +0100
@@ -22,3 +22,5 @@ Type=oneshot
 RemainAfterExit=yes
 ExecStart={{LIBEXECDIR}}/systemd-fsck
 TimeoutSec=infinity
+StandardOutput=null
+StandardError=journal+console
diff -up systemd-stable-255/units/systemd-fsck@.service.in.7~ systemd-stable-255/units/systemd-fsck@.service.in
--- systemd-stable-255/units/systemd-fsck@.service.in.7~	2023-12-08 14:11:50.229849785 +0100
+++ systemd-stable-255/units/systemd-fsck@.service.in	2023-12-08 14:13:23.030789238 +0100
@@ -21,3 +21,5 @@ Type=oneshot
 RemainAfterExit=yes
 ExecStart={{LIBEXECDIR}}/systemd-fsck %f
 TimeoutSec=infinity
+StandardOutput=null
+StandardError=journal+console
