diff -up systemd-stable-254.3/src/boot/efi/meson.build.1~ systemd-stable-254.3/src/boot/efi/meson.build
--- systemd-stable-254.3/src/boot/efi/meson.build.1~	2023-09-09 00:15:31.000000000 +0100
+++ systemd-stable-254.3/src/boot/efi/meson.build	2023-09-13 18:45:26.010781469 +0100
@@ -151,15 +151,15 @@ if get_option('mode') == 'developer' and
 endif
 
 efi_c_ld_args = [
-        # We only support bfd. gold is going away, lld has issues with LTO on x86
+        # We only support bfd. gold is going away, lld has issues with gcc LTO on x86
         # and mold does not support linker scripts.
-        '-fuse-ld=bfd',
+        #'-fuse-ld=bfd', -- commented out because clang LTO isn't affected
 
         '-lgcc',
         '-nostdlib',
         '-static-pie',
         '-Wl,--entry=efi_main',
-        '-Wl,--fatal-warnings',
+        '-Wl,--no-fatal-warnings',
 
         # These flags should be passed by -static-pie, but seem to be missing sometimes.
         '-Wl,--no-dynamic-linker',
@@ -185,6 +185,9 @@ efi_disabled_c_args = cc.get_supported_a
         '-fno-asynchronous-unwind-tables',
         '-fno-exceptions',
         '-fno-unwind-tables',
+        '-U_FORTIFY_SOURCE',
+        '-fno-stack-protector',
+        '-fno-stack-protector-all',
 )
 efi_override_options = [
         'b_coverage=false',
@@ -211,8 +214,8 @@ efi_arch_c_args = {
         'aarch64' : ['-mgeneral-regs-only'],
         'arm'     : ['-mgeneral-regs-only'],
         # Pass -m64/32 explicitly to make building on x32 work.
-        'x86_64'  : ['-m64', '-march=x86-64', '-mno-red-zone', '-mgeneral-regs-only'],
-        'x86'     : ['-m32', '-march=i686', '-mgeneral-regs-only', '-malign-double'],
+        'x86_64'  : ['-m64', '-march=x86-64', '-mno-red-zone', '-mgeneral-regs-only', '-mfpmath=387'],
+        'x86'     : ['-m32', '-march=i686', '-mgeneral-regs-only', '-malign-double', '-mfpmath=387'],
 }
 efi_arch_c_ld_args = {
         # libgcc is not compiled with -fshort-wchar, but it does not use it anyways,
