diff -up systemd-250.20220315/src/boot/efi/meson.build.34~ systemd-250.20220315/src/boot/efi/meson.build
--- systemd-250.20220315/src/boot/efi/meson.build.34~	2022-03-15 20:11:56.242558640 +0100
+++ systemd-250.20220315/src/boot/efi/meson.build	2022-03-15 20:14:47.092083159 +0100
@@ -157,8 +157,7 @@ efi_config_h = configure_file(
         configuration : efi_conf)
 
 efi_cflags = cc.get_supported_arguments(
-        basic_disabled_warnings +
-        possible_common_cc_flags + [
+        [
                 '-fno-stack-protector',
                 '-fno-strict-aliasing',
                 '-fpic',
